lib/tasks/import_datastream.rake

```diff
@@ -4,6 +4,17 @@
 # a class than a block and should be able to handle multiple tasks.
 # rubocop:disable Metrics/BlockLength
 namespace :ssg do
+  desc 'Check if the latest SSG has been synced'
+  task check_synced: [:environment] do
+    message = "SSG datastreams not synced\nDatastream config revision: " \
+      "#{SupportedSsg.revision.inspect}\nDB revision: " \
+      "#{Revision.datastreams.inspect}"
+    if Revision.datastreams != SupportedSsg.revision
+      puts message
+      exit(1)
```

Comment:
This would stop the whole rake process and would not allow to chain this task. Consider using `abort` instead.

Reply:
Using `abort` prevents us from writing tests against this because it aborts the entire test run. I've resorted to using `raise` instead (updates incoming), which we use in other rake tasks as well.

Reply:
Isn't there a mock for that?

Reply:
Aha! I figured it out: `assert_raises SystemExit do` (change incoming)